mempool/src/lib.rs

```diff
@@ -77,6 +84,12 @@ impl<'env> Mempool<'env> {
         {
             let state = self.state.read();
 
+            // Check transaction against rules and blacklist
+            if !state.filter.accepts_transaction(&transaction) || state.filter.blacklisted(&transaction){
```

Comment:
Missing space before `{`.
```suggestion
            if !state.filter.accepts_transaction(&transaction) || state.filter.blacklisted(&transaction) {
```